src/blockMonitor/blockTimeoutDetector.ts

```diff
@@ -16,7 +16,7 @@ export class BlockTimeoutDetector extends StartStopService {
     /**
      * @param timeout The number of milliseconds without a new block before generating
      */
-    constructor(private blockProcessor: BlockProcessor, public readonly timeout: number) {
+    constructor(private blockProcessor: BlockProcessor<any>, public readonly timeout: number) {
```

Comment:
I think you should choose a type here instead of any

Reply:
Changed to IBlockStub.